communication/calendar-guidelines.md

```diff
@@ -142,7 +142,7 @@ Google group: <optional>
 
 > SIG-Contribex: Weekly Marketing Meeting
```

Comment:
We should change this line to say SIG-ContribEx: Weekly Contributor Comms Meeting

Reply:
done